node/tessel-export.js

```diff
@@ -1456,9 +1456,9 @@ class Wifi extends EventEmitter {
   findAvailableNetworks(callback) {
     callback = enforceCallback(callback);
 
-    isEnabled()
-      .then(enabled => {
-        if (enabled) {
+    Promise.all([isEnabled(), isAPEnabled()])
+      .then(enableds => {
```

Comment:
I would change this to `results` and the following `if` statement to be:

```js
if (results[0] || results[1]) {
```

This is a bit more readable and clearer as the `results` of the `Promise.all` resolution.